packages/cli/src/commands/__tests__/lint.test.ts

```diff
@@ -152,11 +156,34 @@ describe('lint', () => {
     expect(writeOutput).toBeCalledWith('<formatted output>', 'foo.json');
   });
 
+  it('writes formatted output to multiple files when using format and output flags', async () => {
+    (formatOutput as jest.Mock).mockClear();
+    (formatOutput as jest.Mock).mockReturnValue('<formatted output>');
+
+    await run(`lint --format html,json --output json:foo.json,html:foo.html ./__fixtures__/empty-oas2-document.json`);
+    await new Promise(resolve => void process.nextTick(resolve));
+    expect(writeOutput).toBeCalledTimes(2);
+    expect(writeOutput).nthCalledWith(1, '<formatted output>', 'foo.html');
+    expect(writeOutput).nthCalledWith(2, '<formatted output>', 'foo.json');
+  });
+
+  it('writes formatted output to multiple files and stdout when using format and output flags', async () => {
+    (formatOutput as jest.Mock).mockClear();
+    (formatOutput as jest.Mock).mockReturnValue('<formatted output>');
+
+    await run(`lint --format html,json --output json:foo.json ./__fixtures__/empty-oas2-document.json`);
+    await new Promise(resolve => void process.nextTick(resolve));
+    expect(writeOutput).toBeCalledTimes(2);
+    expect(writeOutput).nthCalledWith(1, '<formatted output>', undefined);
+    expect(writeOutput).nthCalledWith(2, '<formatted output>', 'foo.json');
+  });
+
```

Comment:
Should we test error cases here? For example:

- `--format json,junit --output results.txt` (output needs to include format)
- `--format json --output junit:results.xml` (unselected format in output)

-

Reply:
Yes, I was waiting to have a feedback on the implementation first